### DISCUSSION TITLE
Best way to iterate on a rewrite?

### DISCUSSION THREAD

**Discussion by marlonschlosshauer** [karthink/gptel#506] [Q&A] [answered]
If I issued a rewrite but don't like the result, to iterate on that I currently do:

1. Reject the rewrite
2. Reselection the region
3. Issue the rewrite with a refined instruction

Before I reject, I see that I can refine my instrutions in the gptel-menu. However, as far as I know I can't issue another rewrite (with the refined instructions) from the menu without first rejecting & reselecting. Or am I missing an option here?

I'm wondering if there is an even faster way. What is your go-to way to iterate on a rewrite?

**Comment by karthink** [comment, marked as answer]
@marlonschlosshauer Iterating on an existing response was built-in, but not indicated explicitly to the user.  I've added the messaging now.  Could you update gptel and try it?  With the cursor in the rewrite overlay, press `RET` or `M-x gptel-rewrite` to see an iterate option.

**Comment by marlonschlosshauer** [comment]
@karthink That did the trick. Thank you!
